components/clearbutton/index.css

```diff
@@ -65,13 +65,20 @@
 		--mod-clear-button-icon-color-down: var(--spectrum-disabled-content-color);
 		--mod-clear-button-background-color: var(--mod-clear-button-background-color-disabled, transparent);
 	}
+
+	&.spectrum-ClearButton--staticWhite {
```

Comment:
So this clear button also gets an [`--overBackground` modifier class](https://github.com/adobe/spectrum-css/blob/a3c858577dcb8c73a18dd5f876ee7029cf80f460/components/clearbutton/stories/template.js#L26) added to it, as opposed to just a `--staticWhite`. I believe we have intentions of deprecating the `--overBackground` modifier in favor of `--staticWhite` to match more of our other static color component variants. Which is great.

My question is if `staticWhite` is sort of the preferred naming, what do we do with the `--overBackground` selector blocks?

Should we add the `&.spectrum-ClearButton--staticWhite` to the [`&:focus-visible` block](https://github.com/adobe/spectrum-css/blob/a3c858577dcb8c73a18dd5f876ee7029cf80f460/components/clearbutton/index.css#L140), now that we have it? Should we add the static white class to the[ block with the token definitions](https://github.com/adobe/spectrum-css/blob/a3c858577dcb8c73a18dd5f876ee7029cf80f460/components/clearbutton/index.css#L49) for `--overBackground`?

Reply:
That makes sense to me! I added `.spectrum-ClearButton--staticWhite` to both blocks that currently have `--overBackground` so we can remove `--overBackground` once it's been deprecated. ✨